doc/Pacemaker_Development/en-US/Ch-Python.txt

```diff
@@ -0,0 +1,81 @@
+= Python Coding Guidelines =
+
+////
+We prefer [[ch-NAME]], but older versions of asciidoc don't deal well
+with that construct for chapter headings
+////
+anchor:ch-python-coding[Chapter 3, Python Coding Guidelines]
+
+== Python Compatibility ==
+
+indexterm:[python,2]
+indexterm:[python,3]
+indexterm:[python,versions]
+
+Pacemaker targets compatibility with python 2.6 and later, and python 3.2 and
+later. It is a good idea to test any changes with both python 2 and 3.
```

Comment:
Good point. I'll add something like, "These versions have added features to be more compatible with each other, allowing us to support both the 2 and 3 series with the same code."

FYI I initially thought we would need to require 2.7, but it turns out 2.6 has everything we need for compatibility purposes. The main thing 2.6 is lacking is the argparse module, which would be a great replacement for our current clunky argument handling. But we can delay the requirement for 2.7 for a while longer, allowing the latest pacemaker to be compiled on older distros such as RHEL 6. argparse is available as a separate package for 6 from EPEL, so we could potentially use it with that dependency, but given that nearly all of our python code is strictly for testing, I don't think an argument overhaul is worth the time right now anyway.